libs/fixtures/.prettierignore

```diff
@@ -0,0 +1 @@
+/**/castVoteRecords/generated
```

Comment:
`libs/cvr-fixture-generator` automatically formats staged JSON files on commit, including generated CVR fixture data. This poses a problem because, post-formatting, the files' hashes will no longer match the hashes used to construct the generated signature file. I _think_ that this is why you ran into auth issues while working on https://github.com/votingworks/vxsuite/pull/4039 @adghayes!

Reply:
Good find!